TAGradingServer/account/admin-gradeable.php

```diff
@@ -1602,10 +1603,108 @@ function moveQuestionUp(question) {
+        var date_submit = Date.parse($('#date_submit').val());
+        var date_due = Date.parse($('#date_due').val());
+        var date_ta_view = Date.parse($('#date_ta_view').val());
+        var date_grade = Date.parse($('#date_grade').val());
+        var date_released = Date.parse($('#date_released').val());
+        var config_path = $('input[name=config_path]').val();
+        var has_space = gradeable_id.includes(" ");
+        var test = /^[a-zA-Z0-9_-]*$/.test(gradeable_id);
+        var unique_gradeable = false;
+        var bad_max_score = false;
+        var check1 = document.getElementById('radio_electronic_file').checked;
+        var check2 = document.getElementById('radio_checkpoints').checked;
+        var check3 = document.getElementById('radio_numeric').checked;
+
+        if (!test || has_space || gradeable_id == "" || gradeable_id === null) {
+            $( "#alert-message" ).dialog( "open" );
+            return false;
+        }
+        if(check1) {
+            if(date_submit < date_ta_view) {
+                alert("DATE CONSISTENCY:  Submission Open Date must be >= TA Beta Testing Date");
+                return false;
+            }   
+            if(date_due < date_submit) {
+                alert("DATE CONSISTENCY:  Due Date must be >= Submission Open Date");
+                return false;
+            }
+            if(config_path == "" || config_path === null) {
+                alert("The config path should not be empty");
+                return false;
+            }
+        }
+        if ($('input:radio[name="ta_grading"]:checked').attr('value') === 'true') {
+            if(date_grade < date_due) {
+                alert("DATE CONSISTENCY:  TA Grading Open Date must be >= Due Date");
+                return false;
+            }
+            if(date_released < date_due) {
+                alert("DATE CONSISTENCY:  Grades Released Date must be >= TA Grading Open Date");
+                return false;
+            }
+        }
+        else {
+            if(check1) {
+                if(date_released < date_due) {
+                    alert("DATE CONSISTENCY:  Grades Released Date must be >= Due Date");
+                    return false;
+                }
+            }
+        }
+        if($('input:radio[name="ta_grading"]:checked').attr('value') === 'true' || check2 || check3) {
+            if(date_grade < date_ta_view) {
+                alert("DATE CONSISTENCY:  TA Grading Open Date must be >= TA Beta Testing Date");
+                return false;
+            }
+            if(date_released < date_grade) {
+                alert("DATE CONSISTENCY:  Grade Released Date must be >= TA Grading Open Date");
+                return false;
+            }
+        }
+        if(!check1 && !check2 && !check3) {
+            alert("A type of gradeable must be selected");
+            return false;
+        }
+
+        var numOfNumeric = 0;
+        var wrapper = $('.numerics-table');
+        var i;
```

Comment:
Put the var inside the for loop for this variable as we're not using i outside of the loop.

(`for (var i = 0; i < $('#numeric_num-items').val(); i++) {`)